### PR TITLE
docs: Clearer instructions for  getting-started/start-a-project

### PR DESCRIPTION
**Pull request by mihado**
* clearer expectation around `agoric start` & `agoric start --reset`
* specify the link to the local wallet endpoint
* specify the link to wallet-ui documentation

### CODE REVIEW

File: main/getting-started/start-a-project.md [hunk 1]
```diff
@@ -101,9 +107,12 @@ in a web browser.
...
 
-Another browser window or tab should appear.
+This should automatically open [http://127.0.0.1:8000](http://127.0.0.1:8000)
+in a new browser window or tab.
 
 ## Use the Dapp to collect your (simulated) tokens
 
-Use the wallet to grant the Dapp's request to connect.
+Use the wallet to grant the Dapp's request to connect. For more information about the wallet UI see
+[https://agoric.com/documentation/guides/wallet/ui.html#wallet-ui](https://agoric.com/documentation/guides/wallet/ui.html#wallet-ui)
```
Comment (dckc) [inline main/getting-started/start-a-project.md:116]:
```suggestion
Use the wallet to grant the Dapp's request to connect. See also the [wallet UI] (/guides/wallet/ui.html#wallet-ui) section.
```

File: main/getting-started/start-a-project.md [hunk 2]
```diff
@@ -101,9 +107,11 @@ in a web browser.
...
 ```
 
-Another browser window or tab should appear.
+This should automatically open [http://127.0.0.1:8000](http://127.0.0.1:8000)
+in a new browser window or tab.
 
 ## Use the Dapp to collect your (simulated) tokens
 
-Use the wallet to grant the Dapp's request to connect.
+Use the wallet to grant the Dapp's request to connect. See also the [wallet UI] (/guides/wallet/ui.html#wallet-ui) section.
```
Comment (dckc) [inline main/getting-started/start-a-project.md:115]: oops: extra space

```suggestion
Use the wallet to grant the Dapp's request to connect. See also the [wallet UI](/guides/wallet/ui.html#wallet-ui) section.
```

**Review by dckc — APPROVED** [review]
thanks for the contribution!

File: main/getting-started/start-a-project.md [hunk 3]
```diff
@@ -101,9 +107,11 @@ in a web browser.
...
 ```
 
-Another browser window or tab should appear.
+This should automatically open [http://127.0.0.1:8000](http://127.0.0.1:8000)
+in a new browser window or tab.
 
 ## Use the Dapp to collect your (simulated) tokens
 
-Use the wallet to grant the Dapp's request to connect.
+Use the wallet to grant the Dapp's request to connect. See also the [wallet UI](/guides/wallet/ui.html#wallet-ui) section.
```
Comment (dckc) [inline main/getting-started/start-a-project.md:115]: The ci bot reminds me:

```suggestion
Use the wallet to grant the Dapp's request to connect. See also the [wallet UI](/guides/wallet/ui.md#wallet-ui) section.
```

Comment (mihado) [reply main/getting-started/start-a-project.md:115]: CI is a great tool! 👍

**Pull request merged**